testsuite/integration/basic/src/test/java/org/wildfly/test/integration/microprofile/metrics/metadata/MicroProfileMetricsRequestMethodsTestCase.java

```diff
@@ -83,7 +87,11 @@ public void checkPost() throws Exception {
 
         try (CloseableHttpClient client = HttpClients.createDefault()) {
             CloseableHttpResponse resp = client.execute(new HttpPost(endpointURL));
-            assertEquals(406, resp.getStatusLine().getStatusCode());
+            assertEquals(405, resp.getStatusLine().getStatusCode());
+            String content = EntityUtils.toString(resp.getEntity());
+            resp.close();
+            assertTrue("'"+ EXPECTED_RESPONSE_MESSAGE + "' message is expected, but was: " + content,
```

Comment:
same as above. Checking HTTP status code should be enough

Reply:
Same here. These tests check https://issues.jboss.org/browse/WFWIP-152. Are you sure you only want to check HTTP status code?